src/tools/brush/ScissorsTool.js

```diff
@@ -0,0 +1,329 @@
+import external from '../../externalModules.js';
+import BaseTool from '../base/BaseTool.js';
+import { setToolCursor } from '../../store/setToolCursor.js';
+
+// Drawing
+import { draw, drawJoinedLines, getNewContext } from '../../drawing';
+import toolColors from '../../stateManagement/toolColors.js';
+
+import {
+  scissorsFillInsideCursor,
+  scissorsEraseInsideCursor,
+  scissorsEraseOutsideCursor,
+  scissorsFillOutsideCursor,
+} from '../cursors';
+
+import store from '../../store';
+import { fillInside, fillOutside } from './utils';
```

Comment:
All the content in `src/tools/brush/utils` should be moved to `src/util/brush`, which already exists.

Reply:
Moved.